_sass/minima/_layout.scss

```diff
@@ -4,41 +4,92 @@
 .site-header {
   border-top: 5px solid $grey-color-dark;
   border-bottom: 1px solid $grey-color-light;
-  min-height: 56px;
+  height: 112px;
 
   // Positioning context for the mobile navigation icon
   position: relative;
+
+  @include media-query($on-smallPhone) {
+    height: 98px;
+  }
+}
+
+.logo-title {
+  position: relative;
+  top: 12px;
+  left: 0;
+
+  @include media-query($on-palm) {
+    top: 10px;
+  }
+  @include media-query($on-smallPhone) {
+    top: 8px;
+  }
 }
+.site-logo {
+  float: left;
+  height: 120px;
+  padding: 0px 5px 0px 0px;
+  margin-left: -6px;
+  margin-top: -5px;
 
+  @include media-query($on-palm) {
+      height: 110px;
+  }
+  @include media-query($on-smallPhone) {
+      height: 100px;
+  }
+}
 .site-title {
-  font-size: 26px;
+  font-size: 70px;
   font-weight: 300;
-  line-height: 56px;
+  line-height: 40px;
   letter-spacing: -1px;
-  margin-bottom: 0;
-  float: left;
+
+  @include media-query($on-palm) {
+    font-size: 51px;
+    line-height: 23px;
+  }
+  @include media-query($on-largePhone) {
+    font-size: 49px;
+    line-height: 23px;
+  }
+  @include media-query($on-smallPhone) {
+    font-size: 44px;
+    line-height: 18px;
+  }
 
   &,
   &:visited {
     color: $grey-color-dark;
   }
+  &:hover {
```

Comment:
I took the underlining specifically out of the title text because I thought that it was was distracting-- the links throughout the rest of the site should still have underlines with mouse hover.  And the title text should still have a pointer change (the cursor changes to a pointing hand) on hover.